treeherder/push_health/utils.py

```diff
@@ -23,7 +28,7 @@ def clean_test(test_name):
         elif clean_name.startswith('http://10.0'):
             left = '/tests/'.join(left.split('/tests/')[1:])
             right = '/tests/'.join(right.split('/tests/')[1:])
-        clean_name = "%s == %s" % (left, right)
+        clean_name = "%s%s%s" % (left, splitter, right)
```

Comment:
There was actually a bug here I hadn't noticed.  It was always putting ` == ` back in, even if it should have been ` != `.

Reply:
You might want to add tests for this module as there's almost no coverage.

Reply:
I'm not sure I follow.  This file has 100% coverage via `tests/push_health/test_utils.py`.  Did you mean a different file?

Reply:
Did you perhaps mean `treeherder/push_health/tests.py` ?  Yeah, I should add a few more tests for that.

Reply:
I was refering to this:
https://codecov.io/gh/mozilla/treeherder/src/master/treeherder/push_health/utils.py

If it should have a higher please let me know!

I don't see `test_utils.py` as one of the tests coverage being collected from:
https://codecov.io/gh/mozilla/treeherder/tree/master/tests/push_health

I don't see it in the output of Travis:
https://travis-ci.org/mozilla/treeherder/jobs/649488955

Reply:
But you're looking at `master`, not this branch.  This branch introduces the tests for that module in `tests/push_health/test_utils.py`.  Are you not able to see the file?  It's the first in the list on the `files changed` tab.

Reply:
My apologies Cam!

I see it now:
https://codecov.io/gh/mozilla/treeherder/pull/5985/src/treeherder/push_health/utils.py?before=treeherder/push_health/utils.py